project/Dependencies.scala

```diff
@@ -42,6 +41,9 @@ object Dependencies {
   val enumeratum: ModuleID =     "com.beachape"               %% "enumeratum"      % "1.5.13"
 
   val javaxServlet: ModuleID = "javax.servlet" % "javax.servlet-api" % javaxServletV % "provided"
+  val jerseyServer: ModuleID = "org.glassfish.jersey.core" % "jersey-server" % "2.26"
```

Comment:
I'd suggest putting the jersey version in a `val` at the top.

Also I'm curious about `glassFishInject` because I don't see it used. Was it copy-paste or do we actually use it somehow?